bundles/org.openhab.binding.livisismarthome/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -389,4 +389,12 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<!-- RebootAction -->
+	<channel-type id="restart">
+		<item-type>Switch</item-type>
+		<label>Restart</label>
+		<description>Restarts the device</description>
+		<category>Switch</category>
```

Comment:
If you'd like to keep the channel stateless, you could set to auto update policy here:
```suggestion
		<category>Switch</category>
		<autoUpdatePolicy>veto</autoUpdatePolicy>
```

Reply:
see above, it isn't working with rules